.github/workflows/tests.yml

```
@@ -0,0 +1,27 @@
+name: runtests
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-version: [1.16.x]
+        platform: [ubuntu-latest]
+    runs-on: ${{ matrix.platform }}
+
+    - uses: actions/setup-go@v2
+      with:
+        go-version: 1.16
```

Make use of the matrix version:

```suggestion
        go-version: ${{ matrix.go-version }}
```